assets/test_iyVesGerm1.yaml

```diff
@@ -8,6 +8,11 @@ dataset:
   HiC:
     reads:
       - reads: /lustre/scratch123/tol/resources/genomeassembly/testdata/iyVesGerm1/hic-arima2/34957_3#2.cram
+  trio:
+    matreads:
+      - https://raw.githubusercontent.com/nf-core/test-datasets/modules/data/genomics/sarscov2/illumina/fastq/test_1.fastq.gz
+    patreads:
+      - https://raw.githubusercontent.com/nf-core/test-datasets/modules/data/genomics/sarscov2/illumina/fastq/test_2.fastq.gz
```

Comment:
As discussed, there is a separate trio test implemented. The 'trio:' section has to be removed here.